packages/flame_spine/example/.gitignore

```diff
@@ -0,0 +1,49 @@
+# Miscellaneous
```

Comment:
I don't think this is needed anymore now when we have a global `.gitignore` (same with the other one)